zoo-css-challenge/index.html

```diff
@@ -1,136 +1,219 @@
-		</section>
-
-		<section class="membership">
-			<h2>Are there any benefits or discounts for members?</h2>
-			<p>There are <strong>AMAZING</strong> benefits of becoming a member with Birmingham Zoo! Our members get special discounts in our shops, cafes, and unlimited entry</p>
-			<button>Learn More</button>
-		</section>
-
-		<section class="programs">
-			<h2>Programs</h2>
-			<div class="card">
-				<h4>Children and Family Programs</h4>
-				<a>Find out More</a>
-			</div>
-			<div class="card">
-				<h4>Adult Programs</h4>
-				<a>Find out More</a>
-			</div>
-			<div class="card">
-				<h4>Youth and young adults</h4>
-				<a>Find out More</a>
-			</div>
-			<h3>Useful Links</h3>
-			<p>
-			</p>
-			<a href="https://www.w3schools.com">w3schools.com</a>
-			<a href="https://www.nationalgeographic.com/" target="_blank">National Geographic</a>
-			<a href="http://www.wikipedia.org" target="_top">Wikipedia</a>
-		</section>
-
- 	</main>
- 	<footer>
- 		<p>Open Monday-Friday 7am-4pm</p><address>1 Birmingham Road</address>
- 	</footer>
+  <head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <img
+        src="https://sharp-villani-e65ba8.netlify.app/zoo.svg"
+        alt="zoo logo"
+        id="logo"
+      />
+      <nav>
+        <ul>
+          <li>Directions</li>
+          <li>Food & Drink</li>
+          <li>Education</li>
+          <li>Your Visit</li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="introduction">
+        <h1>Birmingham Zoo</h1>
+        <p>
+          The zoo is open every day of the year and features three major biomes:
+          the Tropic Zone, Temperate Territory, and the Polar Circle. From
+          tropical birds, to snow leopards, grizzly bears, and one of the
+          nation’s largest colonies of Antarctic penguins, there are animals to
+          enjoy in every season.
+        </p>
+      </section>
+
+      <div id="badge">
+        <p>Opening 2021!</p>
+      </div>
+
+      <section class="bears">
+        <h2>Bears</h2>
+        <p>
+          Did you know that a bear's closest relative is the seal? Or that bears
+          can smell over 2,000 times better than humans?
+        </p>
+        <div class="image-container">
+          <img
+            class="bear1"
+            src="https://placebear.com/200/300"
+            alt="Grizzly Bear"
+            width="200"
+            height="300"
+          />
+          <img
+            class="bear2"
+            src="https://placebear.com/400/300"
+            alt="Grizzly Bear"
+            width="400"
+            height="300"
+          />
+          <img
+            class="bear3"
+            src="https://placebear.com/200/400"
+            alt="Grizzly Bear"
+            width="200"
+            height="400"
+          />
+        </div>
+      </section>
+
+      <section class="tigers">
+        <h2>Tiger Facts</h2>
+
+        <!--<h3>Top 5 Tiger Facts</h3> -->
```

Comment:
Software teams usually don't leave in commented code. We can remove the commented code